collector/drbd_linux.go

```diff
@@ -195,18 +195,18 @@ func (c *drbdCollector) Update(ch chan<- prometheus.Metric) (err error) {
 					metric.isOkay(values[1]), device, "remote")
 			} else if kv[0] == "cs" {
 				// Connection state.
-				var connected float64 = 0
+				var connected float64
 				if kv[1] == "Connected" {
 					connected = 1
 				}
 				ch <- prometheus.MustNewConstMetric(
 					drbdConnected, prometheus.GaugeValue,
 					connected, device)
 			} else {
-				log.Infof("Don't know how to process key-value pair [%s: %s]", kv[0], kv[1])
+				log.Infof("Don't know how to process key-value pair [%s: %q]", kv[0], kv[1])
```

Comment:
Good that you've brought this up. The `/proc` file does expose some tags that we don't process at all. Changing this to `Debugf()`.